homeassistant/components/sabnzbd/__init__.py

```diff
@@ -102,17 +104,20 @@
 
     host = config[CONF_HOST]
     port = config[CONF_PORT]
+    web_root = config[CONF_PATH]
```

Comment:
There's no default in the config schema so we have to use `dict.get` here. Or set a default in the config schema.

Reply:
Sure, will change this to use get :+1:  (a web_root of None passed to pysabnzb lets that library calculate what the root should be)